samples/hello-cube.html

```diff
@@ -0,0 +1,313 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="viewport" content="width=600">
+<title>Web GPU Cube demo</title>
+<script src="gl-matrix-min.js"></script>
+</head>
+<body>
+<canvas height=1000 width=1000></canvas>
+<script>
+/*******
+* This preview of Web GPU uses WHLSL as its shading language. 
+* The choice of shading language(s) that will be ingested by the API is still under deliberation. 
```

Comment:
This is true, but isn't relevant for this particular patch. We're attempting to make a "Hello World" type demo using best practices for WebGPU. The best practices are to use WHLSL. Tests in the test suite will use WHLSL and won't have a disclaimer, so there's no reason to have one here. We should remove the disclaimer.

Reply:
I think the disclaimer is less about the shading language and more about the API for ingesting the shader.